docs/source/models.rst

```diff
@@ -1306,37 +1306,42 @@ Deploy a ``python_function`` model on Microsoft Azure ML
 
 The MLflow plugin `azureml-mlflow <https://pypi.org/project/azureml-mlflow/>`_ can deploy models to Azure ML, either to Azure Kubernetes Service (AKS) or Azure Container Instances (ACI) for real-time serving. 
 
-The resulting container image contains a web server that accepts the following data formats as input:
+The resulting deployment accepts the following data formats as input:
```

Comment:
"format" 
`The resulting deployment accepts the following data format as input:`

Reply:
I got your point, but since it is a bullet list I think plural is right. It also gives the idea that there are other formats that are not supported. Does it make sense?

Reply:
Yep! Makes sense. SGTM.